code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingExtension.java

```diff
@@ -882,6 +898,19 @@ private Map<String, Object> getXDMSharedState(final String extensionName, final
     private boolean eventIsValid(final Event event) {
         return event != null && event.getEventData() != null;
     }
+
+    private void addPushTokenToSharedState(final String pushToken, final Event event) {
```

Comment:
It might be me but `add` here makes me think the push token is being added to an existing shared state. Can you rename to something such as `sharePushToken` or even `createSharedState`?